proxy/http_proxy.go

```diff
@@ -99,8 +99,7 @@ func (p *HTTPProxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if t.RedirectCode != 0 {
-		redirectURL := t.GetRedirectURL(requestURL)
```

Comment:
@magiconair I know this already got merged but I was thinking maybe the above `if` statement should be `t.RedirectCode != 0 && t.RedirectURL != nil` just to be safe?

Reply:
Agreed. See #477